demo/web/index.html

```diff
@@ -185,31 +410,93 @@ <h1>Orca Web Demo</h1>
   onclick="startOrca(document.getElementById('accessKey').value)"
 />
 <hr />
-<label for="text-to-synthesize">Text to synthesize (English)</label>
-<br />
-<textarea id="text-to-synthesize" rows="4" cols="50" disabled maxlength="200">
+<div id="choose-btns" style="display: none">
+  <label for="choose-single-btn">Synthesize speech in one call:</label>
+  <button id="choose-single-btn">
+    Single Synthesize
+  </button>
+  <p><b>OR</b></p>
+  <label for="choose-stream-btn">Synthesize speech live as you type:</label>
```

Comment:
`Synthesize speech from a text stream`

Also, can we change the order of the buttons and have streaming synthesis always be the first option? (also in readmes etc.)